File: output/StarCoder/avatar/Java/Python/atcoder_ABC042_A.py
import sys

for line in sys.stdin:
    x, a, y = map(int, line.split())
    if x == 5 or x == 7 or a == 5 or a == 7 or y == 5 or y == 7:
        if x + y + a == 17:
            print("YES")
        else:
            print("NO")
    else:
        print("NO")

File: output/ChatGPT/avatar/Java/C++/atcoder_ABC042_A.cpp
#include <iostream>
#include <sstream>
using namespace std;

int main() {
    string line;
    getline(cin, line);
    istringstream iss(line);
    int x, a, y;
    iss >> x >> a >> y;
    string h = "NO";
    if (x == 5 || x == 7 || a == 5 || a == 7 || y == 5 || y == 7) {
        if (x + y + a == 17) {
            h = "YES";
        }
    }
    cout << h << endl;
    return 0;
}